### ISSUE TITLE
Support phone portrait mode

### ISSUE THREAD

**Issue by jxu** [city41/stepcharts#6]
The webpage says the page barely fits in landscape mode, just make the UI smaller

using it in landscape mode makes everything too zoomed in anyway

**Comment by city41** [comment]
Yeah this page could be way better on mobile. I plan to improve it next time I'm back working on this project again.
